core/src/main/java/com/netease/arctic/AmoroCatalog.java

```diff
@@ -75,4 +75,9 @@ public interface AmoroCatalog {
    * @throws NoSuchTableException    when table not exists.
    */
   AmoroTable<?> loadTable(String database, String table);
+
+  /**
+   * name of this catalog
+   */
+  String name();
```

Comment:
Who will call this method?

Reply:
Name is the base attribute of catalog, used by PaimonCatalog

Reply:
FormatCatalog needs to be used together with UnifiedCatalog. "name" should be a property of UnifiedCatalog, and discussing FormatCatalog separately doesn't make sense.